ui/src/app/components/admin/audit-page-component.tsx

```diff
@@ -91,7 +85,7 @@ const UserInput = ({initialAuditSubject, auditSubjectType, getNextAuditPath, but
   };
 
   return <React.Fragment>
-    {loadNextSubject && <Navigate to={getNextAuditPath(auditSubject)}/>}
+    {subjectRequested && <Navigate to={getNextAuditPath(auditSubject)}/>}
```

Comment:
this still  seems iffy somehow, to have to evaluate an external variable instead of having this be a prop on Navigate. Other elements have a `show` property, though if we're using an element to describe a state change, we're already well outside my comfort zone. I want to think of elements as nouns or objects, with possible side effects at render time. But having a new class of elements that are not only stateful but  mutate elements around them seems awkward. So something like  `<MyNavigate go={subjectRequested} url={auditUrl}/>`.